forge/README.md

```diff
@@ -37,7 +37,7 @@ You can scope down the policy using:
 3. Resource tagging
 
 > [!NOTE]
-> If you've configured a custom prefix in your Seqera Platform Enterprise Self-Hosted installation, remember to update the resource pattern accordingly.
+> If you've configured a custom prefix for Compute Environments and IAM roles in your Seqera Platform Enterprise Self-Hosted installation, remember to update the resource pattern accordingly.
```

Comment:
```suggestion
> If you've configured a custom prefix for Compute Environments and IAM roles in your Seqera Platform Enterprise self-hosted installation, remember to update the resource pattern accordingly.
```

do we even need to mention that it's self-hosted? aren't enterprise installations all self-hosted?

Reply:
```suggestion
> If you've configured a custom prefix for Compute Environments and IAM roles in your Seqera Platform Enterprise installation, remember to update the resource pattern accordingly.
```